conan/cache/conan_reference_layout.py

```diff
@@ -4,13 +4,21 @@
 from conan.cache.conan_reference import ConanReference
 from conans.errors import ConanException
 from conans.model.manifest import FileTreeManifest
-from conans.paths import BUILD_FOLDER, PACKAGES_FOLDER, SYSTEM_REQS_FOLDER, SYSTEM_REQS, DATA_YML, \
-    rm_conandir, EXPORT_SRC_FOLDER, SRC_FOLDER
-from conans.paths import CONANFILE, SCM_SRC_FOLDER
+from conans.paths import CONANFILE, DATA_YML, rm_conandir
 from conans.util.files import rmdir
 from conans.util.files import set_dirty, clean_dirty, is_dirty
 
 
+# To be able to change them later to something shorter
+SRC_FOLDER = "source"
```

Comment:
We need to make them as short as possible, so they will probably be shortened to "b", "s" or something like that (but later, not in this PR)